VideoLocker/res/drawable/rectangle_with_bottom_border.xml

```diff
@@ -1,13 +1,31 @@
 <?xml version="1.0" encoding="utf-8"?>
-<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
-    <item>
-        <shape android:shape="rectangle">
-            <solid android:color="@android:color/darker_gray" />
-        </shape>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:state_pressed="true">
```

Comment:
not this one, It is applied being applied a LinearLayout that is being added in a ScrollView.